recipes/vprof/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "vprof" %}
+{% set developer = "nvdv" %}
```

Comment:
Would you mind hard coding this? https://github.com/conda-forge/staged-recipes/pull/4734#discussion_r160053925

Reply:
Yes sure! I had deleted the comment and refactored like this. Next commit will be hard coded!